Moonshot/player/States/air.gd

```diff
@@ -16,6 +16,10 @@ onready var jump_delay: Timer = $JumpDelay
 onready var controls_freeze: Timer = $ControlsFreeze
 
 export var acceleration_x := 2500.0
+export var jump_deceleration := 5000.0
+export var air_resistance := 1250.0
```

Comment:
In Godot inspector, you can set setting either of these to **ZERO** will disable each feature separately, so you can test easily with one off, or both off to try the character as it was before.